cl/recap/tasks.py

```diff
@@ -1556,15 +1563,15 @@ def get_attachment_page_by_url(att_page_url: str, court: Court) -> str | None:
 )
 def process_recap_email(
     self: Task, epq_pk: int, user_pk: int
-) -> Optional[Dict[str, Union[int, bool]]]:
+) -> Optional[list[int]]:
     """Processes a recap.email when it comes in, fetches the free document and
     triggers docket alerts and webhooks.
 
     :param self: The task
     :param epq_pk: The EmailProcessingQueue object pk
     :param user_pk: The API user that sent this notification
-    :return: An optional dict to pass to the next task with the docket_pk and a
-     bool True if there was content updated, otherwise False
+    :return: An optional list to pass to the next task with recap documents pks
+     that were downloaded
     """
 
     start_time = now()
```

Comment:
I guess my comment about start_time applies here too?

Reply:
Sure, Here I put it just before adding the docket entries into the DB.